ktxspec.adoc

```diff
@@ -1071,15 +1072,43 @@ have shaders that understand this channel layout.
 The bitstream of the UASTC data is described in Chapter 25 _UASTC
 Compressed Texture Image Format_ of <<KDF13>>.
 
+Basis Universal UASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167). This format supports
+two compression ratios defined by the texel block size in the DFD:
+4x4 and 6x6. Images in this format are a strict RGB-only ASTC HDR
```

Comment:
What's the reference for UASTC HDR 6x6?

Reply:
Not available yet. @richgel999 is working on it.